IFC4x3/PropertyEnumerations/p/PEnum_PositionInTrack.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_PositionInTrack" Name="PEnum_PositionInTrack" UniqueId="eed86143-41d2-40c4-b714-58afc9b7e145">
```

Comment:
Did we not agree on PEnum_RelativePosition used in PositionInTrack pset or property?

Reply:
Yep I missed that...